react-native-purchases-ui/src/index.tsx

```diff
@@ -272,7 +277,10 @@ export default class RevenueCatUI {
         </ScrollView>
         {/*Adding negative margin to the footer view to make it overlap with the extra padding of the scroll*/}
         <InternalPaywallFooterView
-          style={{marginTop: -20}}
+          style={Platform.select({
+            ios: {marginTop: -20},
```

Comment:
iOS works fine so we let React Native handle the sizing

Reply:
Hmm I'm just wondering, where do we add the padding to make it overlap in iOS? We should also test this with several devices like ipad if you haven't already.

Reply:
The padding is in the `ScrollView`, it's set via the `handleSafeAreaInsetsChange`. Is that what you mean? I think I've tested with iPads already, but will double check. In any case, that would be unrelated to this PR I think